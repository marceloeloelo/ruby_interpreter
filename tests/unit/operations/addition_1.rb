0
puts (0)

0.123
puts(0.123)

+1
puts(+1)

+1.23
puts(+1.23)

1+1
puts(1+1)

1+2.3
puts(1+2.3)

1.2+3
puts(1.2+3)

1.2+3.4
puts(1.2+3.4)

"hola " + "hola"
puts("hola " + "hola")
1 + "hola"
puts(1 + "hola")
"hola" + 1
puts("hola" + 1)
