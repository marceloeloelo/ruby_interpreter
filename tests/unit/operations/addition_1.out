0
0.123
1
1.23
2
3.3
4.2
4.6
